src/App.js

```diff
@@ -14,11 +14,96 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    title: ['none'],
+  }
+
+  clear = () => {
+    this.setState({ title: ['none'] });
+  }
+
+  select = (item) => {
+    if (this.state.title[0] === 'none') {
+      this.setState({ title: [] });
+    }
+
+    if (!this.state.title.includes(item)) {
+      this.setState(prevState => ({
+        title: [...prevState.title, item],
+      }));
+    }
+  }
+
+  delete = (item) => {
+    this.setState(prevState => ({
+      title: prevState.title.filter(product => product !== item),
+    }));
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          Selected good:
+          {this.state.title.length !== 0
+            ? this.state.title.join(', ')
+            : this.setState({ title: ['none'] })
+          }
+        </h1>
+        <div className="container">
+          <ul className="container__list">
+            {goodsFromServer.map(product => (
+              <>
+                <li className="container__item">
+                  <span className={
+                    this.state.title.includes(product)
+                      ? 'container__text'
+                      : ''}
+                  >
```

Comment:
```suggestion
                  <span className={this.state.title.includes(product) && 'container__text'}>
```
Or even beter use ClassNames lib [link](https://www.npmjs.com/package/classnames)

Reply:
Дякую, врахую наступного разу